codenaz/react-paginator
# Remove Bootstrap and correct Readme

### What does this PR do?
This PR fixes the bug observed when you import this package and some of your custom styles are overrode because of bootstrap's style in this package and also corrects the exported default package name in the Readme example.


### How should this be manually tested?

- Copy example in the Readme.
- Confirm that no error is given when run.
- Confirm that example runs successfully.
